From a4783ee130e638f51438340b9f8f4de415f9ad76 Mon Sep 17 00:00:00 2001
From: Michal Klocek <michal.klocek@qt.io>
Date: Wed, 8 May 2024 09:07:49 +0200
Subject: [PATCH] Fix dependecy when compiling content/browser

Deals with:

"fatal error: components/spellcheck/spellcheck_buildflags.h:
No such file or directory"

Change-Id: If633fdf9d3d793158ab0dd994c011c17e66c13b5
Upstream-Status: Pending
---
 chromium/content/browser/BUILD.gn | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chromium/content/browser/BUILD.gn b/chromium/content/browser/BUILD.gn
index 1466f33d462..ae459c24d4f 100644
--- a/chromium/content/browser/BUILD.gn
+++ b/chromium/content/browser/BUILD.gn
@@ -334,6 +334,7 @@ jumbo_static_library("browser") {
 
   public_deps = [
     ":accessibility_buildflags",
+    "//components/spellcheck:buildflags",
     "//base/util/memory_pressure",
     "//base/util/type_safety",
     "//ipc",
